Allow examine_env to load hard coded policies

palanc opened the pull request:
Example command: 
```python robohive/utils/examine_env.py -e FrankaReachRandom-v0 -p robohive.utils.examine_env.rand_policy```

vikashplus reviewed (commented):
1. Lets update the DESC to document this use case 
2. Lets add an example as well to show the usage 
3. Lets update the tests/test_examine_env.py to add a unit test 
4. Lets update the wiki/tutorial to add a use case (I can do this)

palanc commented:
Addressed points 1-3. For point 2, could show an example of policy that is not completely random and is likely to succeed once https://github.com/vikashplus/robohive/pull/89 is merged
